### DISCUSSION TITLE
FreeBSD as an installable OS

### DISCUSSION THREAD

**Discussion by zcatav** [quarto-dev/quarto-cli#4480] [Q&A] [answered]
Hi,
I tried to install Quarto on FreeBSD. I can't find any option. Is there a way to install on FreeBSD?
Is there any plan to port to FreeBSD? GNU/Linux, Mac OS and Windows have installer packages but FreeBSD.
Thanks for your attention.

**Comment by mcanouil** [comment]
You could probably build it from source.
Download the source tarball from the release/pre-release you want/need, run `./configure.sh` within its directory (after unarchiving it, where it needs to be on FreeBSD).

Alternatively, you can install the development version <https://github.com/quarto-dev/quarto-cli#development-version>.

**Reply by zcatav** [reply]
Hi,
First tarball option doesn't contain source code only Linux binaries.
Second option for development version actually contains source but also Linux binaries too. All ways end without success due to Linux binaries.

**Reply by mcanouil** [reply]
That's unfortunate.
I do not know much about FreeBSD, and did not see much tools in the wild that support it (_i.e._, being ported to this platform) ...

**Comment by jgenco** [comment]
I wrote a script and a patch that I posted in [this gist](https://gist.github.com/jgenco/c1c2ecb748d3e53f67f8b500ba5d6ceb) that you could try.  I have not done any of the smoke test  yet so I'm not sure what work or does not work.

**Reply by cscheid** [reply]
You'll need the dependencies here: https://github.com/quarto-dev/quarto-cli/blob/main/configuration and Pandoc 3.1 is a requirement (Quarto 1.3 is incompatible with Pandoc 2)

**Reply by jgenco** [reply]
Thanks for confirming Pandoc 3 info. I noticed Pandoc 2.19 failed and Pandoc 3 worked, yet check.ts still said >=2.19.2
https://github.com/quarto-dev/quarto-cli/blob/7e8cf6dc7b1843503f5e433121835975da2e9aee/src/command/check/check.ts#L105-L108

**Reply by cscheid** [reply]
Oh that's a bug, thanks for catching it!

**Comment by hendrack** [comment, marked as answer]
Quarto is now in FreeBSD ports: https://www.freshports.org/textproc/quarto

**Comment by zcatav** [comment]
Thank you for sharing with us.
